### PR TITLE
Add Wonderland Engine SDK

### PR DESCRIPTION
**Pull request by Squareys**
Hi all!

As discussed with @limbofeather, here is the [Wonderland Engine](https://wonderlandengine.com) integration!

Looking forward to your review.

Best,
Jonathan

### CODE REVIEW

File: utils/networking.js [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-import { v4 as uuidv4 } from 'uuid'
 
 // Modify to test a local server
 // const API_BASE = 'http://localhost:2354';
 const API_BASE = 'https://node-1.zesty.market'
 const METRICS_ENDPOINT = API_BASE + '/api/v1/metrics'
 
 const AD_ENDPOINT = 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-rinkeby'
-
-const sessionId = uuidv4();
```
Comment (limbofeather) [inline utils/networking.js:11]: Thanks for removing an appendage for us 🙏🏼

**Review by limbofeather — COMMENTED** [review]
If @msub2 and @Srile give it ✅ then we're good to merge. Thank you @Squareys for the addition!

File: utils/networking.js (hunk 1 above)
Comment (msub2) [reply utils/networking.js:11]: We still need this when we reimplement the metrics, don't we?

Comment (msub2) [reply utils/networking.js:11]: I'd rather just have those lines commented out for now

**Review by msub2 — CHANGES_REQUESTED** [review]
Please restore and just comment out the uuid lines in networking.js so they can be easily restored once we re-implement metrics. Everything else looks good.

File: utils/networking.js (hunk 1 above)
Comment (limbofeather) [reply utils/networking.js:11]: Woops, listen to this man ^

**Comment by Squareys** [comment]
@msub2 Will do! I think I had seen some other function for uuid generation also, though. And this version didn't run in a plain V8-causing a evaluation failure of global space in wonderland-(and possibly not in the browser either?), which was the original reason I found this in the first place.

**Comment by Squareys** [comment]
@msub2 Done, and removed some superfluous formatting changes from that commit also.

**Comment by msub2** [comment]
@Squareys Ah okay, wasn't aware of that V8 issue. When we go to re-implement metrics I'll take another look at it and replace it if necessary.

**Review by msub2 — APPROVED** [review]

**Pull request merged**